apps/backend/src/modules/projects/project.repository.ts

```diff
@@ -1,3 +1,4 @@
+import { NOTHING_DELETED_COUNT } from "~/libs/constants/constants.js";
```

Comment:
Hey, you created a constant for this, while I defined a similar value directly in a function. Which approach do you think is better?

Reply:
I think that global constant (my approach) will be more preferable, since it will be used not only in my and your issues, but in other related to ability to delete something, so this approach will prevent us of creating inplace `ZERO` constant everywhere